azure-pipelines.yml

```diff
@@ -104,11 +145,13 @@ stages:
                   Write-Host "Sonar project version is '${sonarProjectVersion}'"
                   # Set the variable to it can be used by other tasks
                   Write-Host "##vso[task.setvariable variable=SONAR_PROJECT_VERSION]$sonarProjectVersion"
+
```

Comment:
I've inserted empty lines between all the tasks. It's much easier for me to see where a task finishes and another one begins.